backend/app/services/adapters/marketplace_skill.py

```
@@ -0,0 +1,1084 @@
+                Kind.is_active == True,
+            )
+            .first()
+        )
+
+        return marketplace_skill, source_skill
+
+    def get_skill_binary_for_collected(
+        self, db: Session, *, skill: Kind
+    ) -> Optional[bytes]:
+        """Get binary data for a collected skill"""
+        marketplace_skill, source_skill = self.resolve_collected_skill(db, skill=skill)
```

Ruff (0.14.14)

[warning] Unpacked variable `marketplace_skill` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)